internal/app/app.go

```diff
@@ -115,6 +117,7 @@ func setupHandlers(cfg *config.Config, q *queue.Queue[entities.Message], repo re
 	return &handlers.Handlers{
 		VkCallbackHandler: vk_callback_handler.New(q, repo, slog.Default()),
 		Metrics:           metrics_handler.New(cfg.MetricsConfig),
+		Debug:             debug_handler.New(cfg.HttpServerConfig.Host, repo),
```

Comment:
This shouldn't be exposed in production. I think can we keep it now and remove once the database repo is ready. With a database it will be possible to use SQL to populate the repo for manual testing